packages/react-app/src/components/pages/Bounties/index.tsx

```diff
@@ -201,31 +190,46 @@ const SelectExport = ({
 					<MenuItem
 						onClick={ () => {
 							setCsvFormat(BOUNTY_PARCEL_EXPORT_ITEMS);
-							handleCSV(true, function(bounty: any) {
-								if (bounty['claimedBy']) {
-									bounty['compositeName'] = bounty['claimedBy']['discordHandle'];
-								}
-								bounty['compositeName'] += ' - ' + bounty['title'];
-								return bounty;
-							});
+							if (bounties) {
+								const exportBounties = (bounties
+									.filter(({ _id }) => selectedBounties.includes(_id))
+									.filter(bounty => bounty.claimedBy && bounty.payeeData.walletAddress && ALLOWED_CURRENCY_CONTRACTS[bounty.reward.currency as ALLOWED_CURRENCIES_TYPE]) as any[])
```

Comment:
Something about this line doesn't sit right with me. Can you explain it?

Rationale: you're filtering out bounties that don't contain a `claimedBy` `walletAddress` and where the reward currency is incorrect - it sounds to me like there are some validations that need to happen elsewhere?

Then casting to `any` just strikes me as another sign that this filter is doing some of the legwork that another part of the codebase should be handling.

Definitely not a crisis - if it works then cool. But just something to consider

Reply:
The filter only applies on this particular export format - we don't want any Parcel.money fields missing data. Missing wallet address or having a bounty paid in BTC is fine, just not for bounties exported to Parcel